src/Avalonia.Visuals/Media/DrawingContext.cs

```diff
@@ -109,7 +133,14 @@ public void DrawLine(IPen pen, Point p1, Point p2)
         {
             if (PenIsVisible(pen))
             {
-                PlatformImpl.DrawLine(pen, p1, p2);
+                if (pen.IsRenderValid() && p1.IsRenderValid() && p2.IsRenderValid())
+                {
+                    PlatformImpl.DrawLine(pen, p1, p2);
+                }
+                else
+                {
+                    Logger.TryGet(LogEventLevel.Warning)?.Log("DrawLine", this, "Invalid Draw Parameters");
```

Comment:
Why "DrawLine"? It's supposed to be an area

Reply:
fixed